### PR TITLE
Revert changes to staging for performance tests and increased rate limit

### PR DESCRIPTION
**Pull request by richardc0**
- Reverted the Gunicorn worker number to 5 (this should be investigated further on a well baselined system to compare)
- Enabled REDIS
- Increased the rate limit to 400 req/sec as using early testing yesterday 450+ was being achieved so brings it more inline with what can be acheived

### CODE REVIEW

File: manifest-api-staging.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 
 inherit: manifest-api-base.yml
-command: scripts/run_app_paas.sh gunicorn -c /home/vcap/app/gunicorn_config.py --error-logfile /home/vcap/logs/gunicorn_error.log -w 6 -b 0.0.0.0:$PORT application
 services:
```
Comment (venusbb) [inline manifest-api-staging.yml]: Shouldn't you change the worker number rather than deleting the line here?

**Comment by venusbb** [comment]
It was 210000 before the change what do you mean by "using early testing yesterday 450+"? I can see the new change to 240000 is to support 400 request/sec.

**Comment by richardc0** [comment]
The 21000 was an arbitrary amount (a little more than the ~280 req/sec we need to achieve 1 million an hour). The new amount just allows us to test the system with everything turned on (rate limit etc) at that higher rate. (Current load + new load) < 450 req/sec, as that is what SES is set to and I didn't want to top out as the functional tests might not work etc.

File: manifest-api-staging.yml (hunk 1 above)
Comment (servingUpAces) [reply manifest-api-staging.yml]: This is overriding the command line in the manifest-api-base.yml file. Deleting this line means that staging will use the default.

**Pull request merged**